labs/lab-2-javascript-game/elicecilgame.js

```diff
@@ -0,0 +1,59 @@
+var grantHealth = 10;
+var userHealth = 40; 
+var winCount = 0;
+
+// class User {
+//     constructor()
+// }
+
+const startGame = function() {
+    var start = prompt("Would you like to fight the Champion?");
+    if(start === "yes") {
+        name = prompt("Tell us your name.");
+    }
+    startCombat();
+}
+
+const startCombat = function() {   
+    const damageRoll = function() {
+        var damage = Math.round(Math.random() + 1);
+        return damage;
+    }
+    const grantHit = function() {
+        grantHealth = (grantHealth - damageRoll());
+        console.log("Grant is now at", grantHealth, "HP!");  
+    }
+    const userHit = function() {
+        userHealth = (userHealth - damageRoll());
+        console.log(name, "is now at", userHealth, "HP!");
+    }
+    const wins = function() {
+        if (grantHealth === 0) {
+            winCount = (winCount + 1);
```

Comment:
You don't need parens, here -- or around any of the other assignments (lines 23, 27)